profiles/local/smartcard-auth

```diff
@@ -0,0 +1 @@
+auth required pam_debug.so auth=authinfo_unavail
```

Comment:
Why is this file required?

Reply:
If someone enables smartcard auth in gdm using dconf, they will get a proper message.

Reply:
And any authentication attempt will fail of course.